cypress/support/pages/article.pageObject.js

```diff
@@ -0,0 +1,79 @@
+import { generateArticle } from '../generateArticle'; // Імпорт генератора фейкових даних
+import PageObject from '../PageObject';
+
+class articlePageObject extends PageObject {
+  constructor() {
```

Comment:
Ensure that the `PageObject` class is correctly implemented and provides necessary functionalities for the `articlePageObject` class to extend.